first.S

```diff
@@ -0,0 +1,77 @@
+
```

Comment:
@taylorsimpson since you authored this, you can / should add a copyright declaration to this file.  As the copyright holder, you should add your copyright and determine the terms/license associated with this new content.

From our POV I think it'd be ideal for us to accept a BSD-3-clause contribution (marked by an SPDX identifier) to match the rest of this project's content.

Reply:
Added Taylor's copyright and BDS-3 clause license as allowed by him offline.